libyul/backends/evm/AsmCodeGen.cpp

```diff
@@ -146,22 +146,32 @@ pair<shared_ptr<AbstractAssembly>, AbstractAssembly::SubID> EthAssemblyAdapter::
 	return {make_shared<EthAssemblyAdapter>(*assembly), static_cast<size_t>(sub.data())};
 }
 
-void EthAssemblyAdapter::appendDataOffset(AbstractAssembly::SubID _sub)
+void EthAssemblyAdapter::appendDataOffset(std::vector<AbstractAssembly::SubID> const& _subIds)
```

Comment:
```suggestion
void EthAssemblyAdapter::appendDataOffset(vector<AbstractAssembly::SubID> const& _subIds)
```

You have this in a few other `.cpp` files too. There's another one below too.

Reply:
This is still there.